diff -ur actionmailer-1.0.0-o/lib/action_mailer/mail_helper.rb actionmailer-1.0.0/lib/action_mailer/mail_helper.rb
--- actionmailer-1.0.0-o/lib/action_mailer/mail_helper.rb	2004-11-23 18:04:44.000000000 -0700
+++ actionmailer-1.0.0/lib/action_mailer/mail_helper.rb	2005-07-09 14:39:46.000000000 -0600
@@ -1,4 +1,4 @@
-require 'action_mailer/vendor/text/format'
+require 'text/format'
 
 module MailHelper#:nodoc:
   def block_format(text)
@@ -14,4 +14,4 @@
 
     formatted
   end
-end
\ No newline at end of file
+end
diff -ur actionmailer-1.0.0-o/lib/action_mailer.rb actionmailer-1.0.0/lib/action_mailer.rb
--- actionmailer-1.0.0-o/lib/action_mailer.rb	2005-07-06 06:48:14.000000000 -0600
+++ actionmailer-1.0.0/lib/action_mailer.rb	2005-07-09 14:39:29.000000000 -0600
@@ -32,13 +32,11 @@
   end
 end
 
-$:.unshift(File.dirname(__FILE__) + "/action_mailer/vendor/")
-
 require 'action_mailer/base'
 require 'action_mailer/helpers'
 require 'action_mailer/mail_helper'
 require 'action_mailer/quoting'
-require 'action_mailer/vendor/tmail'
+require 'tmail'
 require 'net/smtp'
 
 ActionMailer::Base.class_eval do
